cornice_swagger/swagger.py

```diff
@@ -104,11 +111,26 @@ def _build_paths(self, ignore=['head', 'options'], default_tags=None,
             path = self._extract_path_from_service(service, **kwargs)
 
             for method_name, view, args in service.definitions:
-                if method_name.lower() in ignore:
+
+                if method_name.lower() in ignore_methods:
                     continue
 
                 op = self._extract_operation_from_view(view, args, **kwargs)
 
+                if ignore_ctypes and 'consumes' in op:
+                    if set(op.get('consumes')).issubset(set(ignore_ctypes)):
```

Comment:
I could be wrong but this formulation seems more natural/correct to me. The current behavior will ignore both views, right? That doesn't seem right to me, since the documentation specifically refers to `content-types that should NOT be presented`. I'd expect the first view to be presented, but with content types `['application/json']`.